main.py

```diff
@@ -30,13 +31,14 @@ def create_account(self, account_type: str = "Normal", initial_balance: float =
         else:
             raise ValueError("Invalid account type.")
 
-        customer = Customer(account_number, account)
+        return account
+
+    def add_customer(self, customer: Customer):
```

Comment:
You could condsider specifying the return type as None, given that all other functions include the return type